journal-letters/modernized/04-chapter-1.adoc

```diff
@@ -926,8 +1002,8 @@ with desires that you may witness a renewal of strength
 to persevere in the path of peace,
 by looking to the Lord for help, who is the never-failing helper of his people.
 
+[.signed-section-closing]
 I remain in near affection, your loving cousin,
 
+[.signed-section-signature]
 J+++.+++ K.
-
-Colchester, 3rd 10th Month, 1763.
```

Comment:
Did this line get accidentally deleted?

Reply:
It actually belonged to the next letter that begins on chapter two so I moved it over to that.

Reply:
ahh, that makes sense. thanks, merging now. 👍